appIcons.js

```diff
@@ -1184,7 +1189,8 @@ const DockAppIconMenu = class DockAppIconMenu extends PopupMenu.PopupMenu {
             }
 
             if (Shell.AppSystem.get_default().lookup_app('org.gnome.Software.desktop') &&
```

Comment:
Does this still apply? I mean: with the new Snap Store, there is nothing with `org.gnome.Software` at/in `/var/lib/snapd/desktop/applications`...

Reply:
That's the code for non-snaps, dash-to-dock it's not an ubuntu project so we definitely support also default GNOME installations where GNOME software is installed.

Reply:
Oh, ok. Understood.